app/vacancy_sources/fusion_vacancy_source.rb

```diff
@@ -0,0 +1,89 @@
+class FusionVacancySource
+  FEED_URL = ENV.fetch("VACANCY_SOURCE_FUSION_FEED_URL").freeze
+  SOURCE_NAME = "fusion".freeze
+
+  class FusionImportError < StandardError; end
+
+  include Enumerable
+
+  def self.source_name
+    SOURCE_NAME
+  end
+
+  def each
+    results.each do |result|
+      v = Vacancy.find_or_initialize_by(
+        external_source: SOURCE_NAME,
+        external_reference: result["reference"],
+      )
+
+      # An external vacancy is by definition always published
+      v.status = :published
+      # Consider publish_on date to be the first time we saw this vacancy come through
+      # (i.e. today, unless it already has a publish on date set)
+      v.publish_on ||= Date.today
+
+      v.assign_attributes(attributes_for(result))
+
+      yield v
+    rescue StandardError => e
+      Sentry.capture_exception(e)
+    end
+  end
+
+  private
+
+  def attributes_for(item)
+    {
+      job_title: item["jobTitle"],
+      job_advert: item["jobAdvert"],
+      salary: item["salary"],
+      expires_at: Time.zone.parse(item["expiresAt"]),
+      external_advert_url: item["applicationUrl"],
+      job_role: item["jobRole"].presence&.gsub("leadership", "senior_leader")&.gsub(/\s+/, ""),
+      ect_status: ect_status_for(item),
+      subjects: item["Subjects"].presence&.split(","),
+      working_patterns: item["workingPatterns"].presence&.split(","),
+      contract_type: item["contractType"].presence,
+      phases: item["phase"].presence&.parameterize(separator: "_"),
+
+      # TODO: What about central office/multiple school vacancies?
+      job_location: :at_one_school,
+      readable_job_location: organisations_for(item).first&.name,
+      organisations: organisations_for(item),
+      about_school: organisations_for(item).first&.description,
+    }
+  end
+
+  def ect_status_for(item)
+    return unless item["ect_suitable"].presence
+
+    item["ect_suitable"] == "yes" ? "ect_suitable" : "ect_unsuitable"
+  end
+
+  def organisations_for(item)
+    [school_group(item).schools.find_by(urn: item["schoolUrns"].first)]
+  end
```

Comment:
Yeah it's something we need to define with the ATS itself. 
There will be vacancies that can be applied to multiple school within the same Trust. What do we do in that case?

Once we have the answer this can be refactored a bit. But good spot!